drivers/overlay/overlay.go

```diff
@@ -709,7 +709,7 @@ func (d *Driver) Get(id, mountLabel string) (_ string, retErr error) {
 		return "", err
 	}
 
-	if err := os.Chown(path.Join(workDir, "work"), rootUID, rootGID); err != nil {
+	if err := os.Chown(path.Join(workDir, "work"), int(rootUID), int(rootGID)); err != nil {
```

Comment:
should we raise an error when rootUID is bigger than MaxInt.  On 32 bits systems this will be otherwise casted without any indication to the user

Reply:
probably wouldn't hurt to at least do a log.debug.